### PR TITLE
add design to post show page

### LINKED ISSUE #29: Design post pages
Create designs for the views realated to Post pages

### PR DESCRIPTION
**Pull request by SepsiLaszlo**
### What's new
* new design for the post show page
* implement design for the post edit page 

airsoft-show

### CODE REVIEW

File: app/javascript/packs/album.js [hunk 1]
```diff
@@ -1,48 +0,0 @@
...
-  })
-})
-
-var content = document.querySelector("#content")
-var masonry = new Masonry(content, {
-  percentPosition: true
-})
-imagesLoaded(content, (instance) => {
-  masonry.layout();
-})
```
Comment (bbucsy) [inline app/javascript/packs/album.js]: I am not entirely sure why did you delete `album.js` but it kind of breaks the gallery system

File: app/views/posts/edit.html.erb [hunk 2]
```diff
@@ -1,6 +1,10 @@
-<h1>Editing Post</h1>
+<div class="post-container mt-2">
+  <div class="button-collection mb-2">
+    <div class="d-flex justify-content-between">
+      <%= link_to 'Vissza', @post,  class: "btn btn-success" %>
+    </div>
+  </div>
+  <h1 class="d-flex justify-content-center mb-2">Poszt szerkesztése</h1>
```
Comment (bbucsy) [inline app/views/posts/edit.html.erb]: If would be more consistent if you change the text to Hungarian in the 'new post' page too

File: app/javascript/stylesheets/post.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
+
+.post-container{
```
Comment (bbucsy) [inline app/javascript/stylesheets/post.scss]: As we are using bootstrap, it would be wiser to use the built in grid system instead of the `@media` rule.
Also you there is a really nice bootstrap component that has almost everything in this scss file: [card](https://getbootstrap.com/docs/5.0/components/card/)

I am not really good in front-end work, so we should consult @triszt4n  on this matter.

**Review by bbucsy — CHANGES_REQUESTED** [review]

File: app/javascript/packs/album.js (hunk 1 above)
Comment (SepsiLaszlo) [reply app/javascript/packs/album.js]: It was a mistake, thanks for mentioning.

File: app/views/posts/edit.html.erb (hunk 2 above)
Comment (SepsiLaszlo) [reply app/views/posts/edit.html.erb]: Done!

File: app/javascript/stylesheets/post.scss (hunk 3 above)
Comment (SepsiLaszlo) [reply app/javascript/stylesheets/post.scss]: I think it is simpler to use a media query here.
I think the card component is not exactly for this application.

@triszt4n what do yout think?

**Review by SepsiLaszlo — COMMENTED** [review]
I made the requested changes, removed the custom css and implemented the views with bootstrap classes.
airsoft-show

**Review by bbucsy — APPROVED** [review]
Looks really good. I think its more manageable this way, with Bootstrap's built in components.
